### PR TITLE
Theme setup

### PR DESCRIPTION
**Pull request by Eullin**
theme file

### CODE REVIEW

File: src/theme/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import colors from './colors'
```
Comment (hexjelly) [inline src/theme/index.ts]: I like keeping colors in separate file! 👍

File: src/theme/index.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      borders: [
+        0,
+        '1px solid',
+        '2px solid',
+        '3px solid',
+        '4px solid',
+        '5px solid',
+        '6px solid',
+      ],
+      radius: [3, 4, 5, 10, 20, 30, 60, 120, '50%'],
```
Comment (hexjelly) [inline src/theme/index.ts]: Do you know if this works with styled-system? I saw in theme docs that they have field called `radii`, but maybe there is also some alias for this? 😄

File: src/theme/index.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      widths: [36, 40, 45, 48, 54, 70, 81, 128, 256],
+      heights: [36, 40, 44, 48, 50, 55, 70, 80, 120, 230],
+      maxWidths: [16, 32, 64, 128, 256, 512, 768, 1024, 1536],
+      colors,
+}
```
Comment (hexjelly) [inline src/theme/index.ts]: We should add `space` field with some values for padding, margin, etc. Array with 0 as first value is good default, and then some sensible scaling after that? Like 0,1,2,4,8,16,32 for example. But could be anything.

**Comment by hexjelly** [comment]
Ooh, also! What about adding something like this [https://styled-system.com/theme-specification/#scale-aliases](https://styled-system.com/theme-specification/#scale-aliases)? We could then have named font sizes for like general text, heading, etc

File: src/theme/index.ts (hunk 2 above)
Comment (Eullin) [reply src/theme/index.ts]: Just checked, should be radii, I'll fix it.

File: src/theme/index.ts (hunk 3 above)
Comment (Eullin) [reply src/theme/index.ts]: We should indeed!

**Comment by Eullin** [comment]
Yes, also for breakpoints. I'll work on these.

**Comment by Eullin** [comment]
fixes #7

**Pull request merged**